### PR TITLE
feat: support drawings and clear graphics

### PR DESCRIPTION
**Pull request by xuew8412**
Updates for 1.16:
- Support add/remove/clear drawing interfaces
- Support clear graphics interface

### Review and Test:
- [x] Please help to review @yong7743

### CODE REVIEW

File: src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs [hunk 1]
```diff
@@ -364,6 +365,116 @@ public static async Task<string> RemoveGraphic(string graphicId)
...
+                return await GetResponseContent(responseMessage);
+            }
+            catch (Exception ex)
+            {
+                return ex.Message;
+            }
+        }
+
+        /// <summary>
+        /// Add a  to Earth.
```
Comment (yong7743) [inline src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs:388]: Dropped a word here?

Comment (xuew8412) [reply src/ArcGISEarth.AutoAPI.Shared/AutomationAPIHelper.cs:388]: Ahhh...

**Review by yong7743 — APPROVED** [review]
Looks good.

**Comment by yong7743** [comment]
Merging it since 1.16 is live.

**Pull request merged**